source/getting-started.txt

```diff
@@ -0,0 +1,63 @@
+.. _app-services-get-started:
+
+================
+Getting Started
+================
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+   
+   Template Apps </template-apps>
+   Tutorials </tutorials>
+
+The resources on this page are designed to help you explore Atlas App 
+Services. Visit the Template Apps page to bootstrap an App Services app. 
+Check out tutorials to learn more about features, such as Device Sync 
+and client authentication.
+
+Template Apps
+-------------
+
+Bootstrap App Services with a Template App. Template apps bring together
+many of the building blocks available in Atlas App Services and start you 
+off with a prebuilt app that you can customize. `Learn more about the Template
+Apps here. <https://www.mongodb.com/docs/atlas/app-services/template-apps/>`_
+
+
+Tutorials
+---------
+
+Follow along with a tutorial to learn how to use App Services.
+
+.. card-group::
+   :style: extra-compact
+   :columns: 3
+
+   .. card::
+      :headline: Build on a Template App with Device Sync
+      :url: https://www.mongodb.com/docs/realm/tutorial/
+      :icon: /images/icons/realm/mobile-sync.svg
+      :icon-alt: Sync Icon
+      
+      Learn how to use Atlas Device Sync with the Realm client SDKs. 
```

Comment:
I think this description isn't quite accurate, as the tutorials don't teach you how to "use Atlas Device Sync", but they show how to modify the template app to sort of "build your own" - and along the way, you learn important sync concepts. I think we can tweak it a bit to emphasize that aspect, but I'm definitely open to other wording.
```suggestion
      Learn how to develop apps with Realm client SDKs that sync data seamlessly using Device Sync.
```